### DISCUSSION TITLE
Welcome to weatherApp-Reactjs Discussions!

### DISCUSSION THREAD

**Discussion by gauravghai** [gauravghai/weatherApp-Reactjs#4] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by SohamRoy-01** [comment]
#### Hi, @gauravghai myself Soham Roy
* I am not from It/Cs background My major is in civil then I have a job but I resign it bcz I didn't want to continue in that field..
* I really enthusiastic about full stack development currently learning JavaScript after I will move to react & redux ..
* I want to be a frontend Dev ..
* In the community I contribute to many open sources projects and helps grow myself to move further with the community.
* I watched your YouTube videos its really motivating that how u showcase every bit of knowledge with us.
* Really happy to join the community..
